robot/impl/local_robot_test.go

```diff
@@ -4284,3 +4284,71 @@ func TestMaintenanceConfig(t *testing.T) {
 		test.That(t, sensorUnBlocked, test.ShouldNotBeNil)
 	})
 }
+
+func TestRemovingOfflineRemote(t *testing.T) {
```

Comment:
I think the test goes really quickly so the complete config loop doesn't have a chance to try and reconnect before the reconfigure removes the resources. There is a chance that this test could flake in the future. if the complete config loop processes the remote before the reconfig removes the resources then the test will hang and time out waiting to connect to the remote.

I am in favor of removing this test since TestRemovingOfflineRemotes covers our testing case. Also it is a way safer test with how we manage the go routines we setup

Reply:
I actually think it is because we don't go through completeConfigForRemotes if the remote is offline - it doesn't show up as unhealthy and it wouldn't be reconfigured. I think there's value in keeping the test if we can make sure things are safe